test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -2233,6 +2234,10 @@
     (ca/overwrite-existing-crl! crl rest-of-full-chain (:capub settings) (:cakey settings) ca-cert (vec all-serials) (:cacrl settings))
     (let [[updated-crl & _rest-of-full-chain] (utils/pem->crls (:cacrl settings))
           updated-crl-number (utils/get-crl-number updated-crl)]
+      ;; currently the CA certs only have one cert, and not an intermediate
+      ;; so while this test is a valid test, it is very unlikely to fail.
```

Comment:
It looks like there's a "bundle-cadir" that has a small CA chain in it that we can use. See usage here: https://github.com/puppetlabs/puppetserver/pull/2791/files#diff-5dee42180e003e4d76fbca7cddd079fe43ce975631592037eab3d19bf04af052R489